spec/ddtrace/contrib/shoryuken/tracer_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+require 'spec/ddtrace/contrib/shoryuken/shoryuken_spec_helper'
+
+RSpec.describe Datadog::Contrib::Shoryuken::Tracer do
+  let(:worker) { TestWorker.perform_async(body) }
+  let(:tracer) { ::Datadog::Tracer.new(writer: writer) }
+  let(:writer) { FauxWriter.new }
+  let(:sqs_queue) { double Shoryuken::Queue, visibility_timeout: 60 }
+  let(:queue) { 'default' }
+  let(:body) { 'test' }
+  let :sqs_msg do
+    double \
+      Shoryuken::Message,
+      queue_url: queue,
+      body: 'test',
+      attributes: {
+        id: 1,
+        test_attribute: 'test'
+      }
+  end
+
+  before do
+    allow(Shoryuken::Client).to receive(:queues).with(queue).and_return sqs_queue
```

Comment:
@delner maybe do you know or some one can suggest right way how to run shoryuken client for test to getting spans from writer